hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/om/ratis/OMRatisSnapshotInfo.java

```diff
@@ -98,13 +98,16 @@ private RatisSnapshotYaml readRatisSnapshotYaml() throws IOException {
 
   /**
    * Update and persist the snapshot index and term to disk.
-   * @param index new snapshot index to be persisted to disk.
+   * @param lastAppliedTermIndex new snapshot index to be persisted to disk.
    * @throws IOException
    */
-  public void saveRatisSnapshotToDisk(long index) throws IOException {
-    updateSnapshotIndex(index);
+  public void saveRatisSnapshotToDisk(TermIndex lastAppliedTermIndex)
+      throws IOException {
+    updateTermIndex(lastAppliedTermIndex.getTerm(),
+        lastAppliedTermIndex.getIndex());
     writeRatisSnapshotYaml();
-    LOG.info("Saved Ratis Snapshot on the OM with snapshotIndex {}", index);
+    LOG.info("Saved Ratis Snapshot on the OM with snapshotIndex {}",
+        lastAppliedTermIndex);
   }
```

Comment:
updateSnapshotIndex() function is no longer used. We can remove it.

Reply:
removed it.